Makefile.os2

```diff
@@ -158,6 +158,9 @@ SDL_hidapi.obj: SDL_hidapi.c
     wcc386 $(CFLAGS_DLL) -za99 -fo=$^@ $<
 !endif
 
+yuv_rgb_sse.obj: yuv_rgb_sse.c
+    wcc386 $(CFLAGS_DLL) -wcd=202 -fo=$^@ $<
```

Comment:
@sezero This was needed because `yuv_rgb_sse.c` has commented-out code that causes a static variable to not get used.